src/main/java/gov/nj/innovation/customAwsIdp/SamlGenerator.java

```diff
@@ -49,7 +50,7 @@
 public class SamlGenerator {
 
     /* Constants */
-    private static final Logger logger = Logger.getLogger(SamlGenerator.class);
+    private static final Logger logger = LogManager.getLogger(SamlGenerator.class);
```

Comment:
This `Logger.getLogger(...` -> `LogManager.getLogger(...` change is part of the upgrade

Reply:
Notice above how the imports change from:
`org.apache.log4j...` to:
`org.apache.logging.log4j...`

This is a really annoying way they try to be backwards compatible (I guess?) but upgrade their code >.<